conda_forge_tick/migrators/cross_compile.py

```diff
@@ -23,7 +23,7 @@ class UpdateConfigSubGuessMigrator(MiniMigrator):
     post_migration = True
 
     def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
-        build_reqs = attrs.get("meta_yaml", {}).get("requirements").get("build")
+        build_reqs = attrs.get("requirements"{}).get("build", set())
```

Comment:
```suggestion
        build_reqs = attrs.get("requirements", {}).get("build", set())
```
Sorry! I missed a comma